pkg/controller/config/config_controller.go

```diff
@@ -323,7 +323,19 @@ func (r *ReconcileConfig) applyAddons(req reconcile.Request, cfg *op.Config) (re
 	log.Info("successfully applied all addon resources")
 
 	err = r.updateStatus(cfg, op.ConfigCondition{Code: op.AppliedAddons, Version: flag.TektonVersion})
-	return reconcile.Result{Requeue: true}, err
+	if err != nil {
+		return reconcile.Result{}, err
+
+	}
+
+	labels := r.addons.Filter(mf.ByKind("Deployment"), mf.ByName(flag.TriggerControllerName)).Resources()[0].GetLabels()
```

Comment:
could you move this into a function which accepts, deploymentName and label name as args

Reply:
make sense :D

Reply:
resolved